src/components/Home.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { LoremIpsum } from 'react-lorem-ipsum';
+import styles from '../styles/Home.module.css';
```

Comment:
- [ ] Well done styling the app 🥇🥇 I appreciate your effort for using CSS modules to add styles, however, please make sure to follow the project requirements and use **_React Basic Styling_** for this milestone.  Following is a quote for a related requirement. I know you can fix this ✨

  > _In this project, you should use React basic styling._


**_Note: Please apply this to all files_**